packages/tools/generator-component/generators/app/templates/test/specs/component/f-skeleton.component.__spec__.js

```diff
@@ -1,6 +1,12 @@
 import <%= name.filename %>Component from '../../../test-utils/component-objects/f-<%= name.class %>.component';
 
 describe('f-<%= name.class %> component tests', () => {
+    beforeEach(() => {
+        browser.url('?path=<%= storybook.path %>');
```

Comment:
This is currently coming out in camel case when it should be hyphenated to match the generated storybook page

Reply:
`?path=/story/components-molecules--media-element-component` vs `?path=story/components-molecules--mediaElement`

Reply:
first is the storybook page

Reply:
👍 Will fix

Reply:
Done

Reply:
still not quite working `?path=story/components-molecules--content-banner` should be `?path=/story/components-molecules--content-banner-component`. Other than this happy to merge

Reply:
Have fixed this now